service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/documents/ManageDocumentsControllerAboutToSubmitTest.java

```diff
@@ -133,7 +133,17 @@ void shouldPopulateFurtherDocumentsCollection() {
 
     @Test
     void shouldPopulateCorrespondenceEvidenceCollection() {
-        List<Element<SupportingEvidenceBundle>> furtherEvidenceBundle = buildSupportingEvidenceBundle();
+        List<Element<SupportingEvidenceBundle>> furtherEvidenceBundle = List.of(
```

Comment:
you can use 
wrapElements(
            SupportingEvidenceBundle.builder()
                .name("test1")
                .dateTimeUploaded(now().minusDays(2))
                .uploadedBy(USER)
                .build(),
            SupportingEvidenceBundle.builder()
                .name("test2")
                .dateTimeUploaded(now())
                .uploadedBy(USER)
                .build());

Reply:
updated the test